src/TextField/TextField.js

```diff
@@ -356,6 +361,14 @@ class TextField extends Component {
     }
   };
 
+  handleComposition = (event) => {
+    switch (event.type) {
+      case 'compositionupdate':
+      case 'compositionstart': this.setState({isTyping: true}); break;
+      case 'compositionend': this.setState({isTyping: false}); break;
+    }
+  };
+
```

Comment:
You also need to pass the event though to the appropriate composition prop if set. There are a couple of patterns for that, in use,  either test if the prop is defined or (my preference) give the props empty arrow function defaults: `() = {}`.